README.md

```diff
@@ -15,51 +15,96 @@ When you are done:
 #### Loops & Lists
 - [Max Value](https://repl.it/@Admin7/maxvalue)
 ```python
-# copy & paste your solution here
+def return_max(l):
+  result = l[0]
+  if len(l) == 0:
+    return None
+  for i in l:
+    if i > result:
+      result = i
+  return result
 ```
 
 - [Min Value](https://repl.it/@Admin7/minvalue)
 ```python
-# copy & paste your solution here
+def return_min(l):
+  if len(l) == 0:
+    return None
+  result = l[0]
+  for i in l:
+    if i < result:
+      result = i
+  return result
 ```
 
 - [Extend](https://repl.it/@Admin7/extendlist)
 ```python
-# copy & paste your solution here
+def return_extended(l, iter):
+  for i in iter:
+    l.append(i)
+  return l
 ```
 
 - [Extend](https://repl.it/@Admin7/creatematrix)
 ```python
-# copy & paste your solution here
+def create_matrix(h, w):
+  result = []
+  for i in range(h):
+    row = []
+    for x in range(w):
+      row.append('#')
+    result.append(row)
+  return result
 ```
 
 
 #### Dictionaries
 - [Receipt Tally](https://repl.it/@Admin7/receipttally)
 ```python
-# copy & paste your solution here
+def get_total(trans, category):
+  total = 0
+  for dict in trans:
+    if dict['category'] == category:
+      total = total + dict['subtotal']
+  return total
 ```
 
 - [Translation](https://repl.it/@Admin7/translations)
 ```python
-# copy & paste your solution here
+def translation(word, lang):
+  for key in translations:
+    print(key)
+    if key == word:
+      return translations[word][lang]
```

Comment:
This probably gives you an error, right? Because one of the test cases looks for `''` for the language `''`, neither of which exist. So we need to write explicit error-checks for:
1. When the dictionary doesn't have the word
2. When the word doesn't have the language

With that in mind, how would you rewrite this?